package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@wrhs/diagrams",
+  "version": "1.0.0",
+  "description": "Utility tool to generate mermaid diagrams for Warehouse.ai",
+  "main": "index.js",
+  "bin": {
+    "wrhs-diagrams": "./index.js"
+  },
+  "scripts": {
+    "test": "mocha test/**/*.js",
```

Comment:
put the path to files in quotes so it uses glob rather than the OS to evaluate it.

```
"test": "mocha \"test/**/*.js\"",
```